src/main/webapp/app/course/learning-paths/components/learning-path-student-nav-overview/learning-path-student-nav-overview.component.html

```diff
@@ -0,0 +1,33 @@
+<div>
+    <div class="row justify-content-between align-items-center w-100 m-0 p-3">
+        <span jhiTranslate="artemisApp.learningPath.navigation.overviewTitle" class="col-md-auto p-0 fw-bold"></span>
+        <!--        TODO: Include in follow up PR to show competency dependencies -->
+        <!--        <div class="col-md-auto p-0">-->
+        <!--            <button type="button" class="btn btn-outline-secondary">Dependency graph</button>-->
+        <!--        </div>-->
+    </div>
+    <hr class="m-0" />
+    @if (isLoading()) {
+        <div class="row justify-content-center m-4">
+            <div class="spinner-border text-primary" role="status">
+                <span class="sr-only">Loading...</span>
+            </div>
+        </div>
+    } @else {
+        @for (learningObject of learningObjects(); track learningObject; let i = $index) {
+            <div
+                (click)="selectLearningObject(learningObject)"
+                class="row px-3 py-3 m-0"
+                [ngClass]="{ 'bg-light': isEqualToCurrentLearningObject(learningObject.id, learningObject.type), clickable: isLearningObjectSelectable(learningObject) }"
```

Comment:
I had another look at the change detection with signals. These function calls will not invoke change detection. Additionally when I do the checks in the HTML it will be very cluttered and confusing